aderyn_core/src/audit/investigators.rs

```diff
@@ -0,0 +1,208 @@
+//! This module helps with strategies on performing different types of investigations.
```

Comment:
Do you think we should move this to a different folder? Maybe `browser` or `context` or its own new one?

Reply:
`browser` would suitable.